pkg/ebpf/events_processor.go

```diff
@@ -340,8 +345,42 @@ func (t *Tracee) processEvent(event *trace.Event) error {
 
 	case InitModuleEventID:
 		t.invokeIoctlTriggeredEvents()
+	case DetectHookedProcFopsEventID:
+		fopsAddresses, err := getEventArgUlongArrVal(event, "hooked_fops_pointers")
+		if err != nil || fopsAddresses == nil {
+			return fmt.Errorf("error parsing detect_hooked_proc_fops args: %w", err)
+		}
+		hookedFops := make([]bufferdecoder.HookedSymbolData, 0, 0)
+		for idx, addr := range fopsAddresses {
+			inTextSeg, err := t.kernelSymbols.TextSegmentContains(addr)
+			if err != nil {
+				return fmt.Errorf("error checking kernel address: %v", err)
+			}
+			if !inTextSeg {
+				hookingFunction, err := t.kernelSymbols.GetSymbolByAddr(addr)
+				if hookingFunction.Owner == "system" && err == nil {
```

Comment:
BTW,
I think that this if-if-else form is not a best practice.
if recommend you to change it to be:
```suggestion
if err == nil {
    if hookingFunction.Owner == "system" {
    } else {}
} else {}
```